### PR TITLE
Development wordpress

### PR DESCRIPTION
**Pull request by 0oM4R**
### Description

WordPress solution,

### Changes

add WordPress to the weblet

### Related Issues

 - #1228 
 - https://github.com/threefoldfoundation/info_manual3/issues/158
 - https://github.com/threefoldtech/info_grid/pull/3
 - https://github.com/threefoldtech/tf-images/pull/125
### Checklist

- [x] Build pass
- [x] Documentation
- [x] Code format and docstrings

### CODE REVIEW

**Comment by 0oM4R** [comment]
- [x] Replace documentation link 
- [x] Replace flist with the official one

File: src/elements/wordpress/wordpress.wc.svelte [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+    profile = detail;
+  }}
+/>
+<div style="padding: 15px;">
+  <form class="box" on:submit|preventDefault={deployWordpressHandler}>
+    <h4 class="is-size-4 mb-4">Deploy a Wordpress Instance</h4>
+    <p>
+      WordPress is the simplest, most popular way to create your own website or blog. In fact, WordPress powers over
+      43.3% of all the websites on the Internet.
+      <a target="_blank" href="https://manual.grid.tf/weblets/weblets_wordpress.html"> Quick start documentation</a>
```
Comment (zaelgohary) [inline src/elements/wordpress/wordpress.wc.svelte:133]: Manual link does not work.

File: src/App.svelte [hunk 2]
```diff
@@ -21,7 +21,7 @@
   import ContractsList from "./elements/ContractsList/ContractsList.wc.svelte";
   import Algorand from "./elements/algorand/Algorand.wc.svelte";
   import QVM from "./elements/qvm/Qvm.wc.svelte";
-
+  import wordpress from "./elements/wordpress/wordpress.wc.svelte";
```
Comment (zaelgohary) [inline src/App.svelte]: Capitalize wordpress as it's been done in all solutions.

**Review by zaelgohary — CHANGES_REQUESTED** [review]

File: src/elements/wordpress/wordpress.wc.svelte (hunk 1 above)
Comment (0oM4R) [reply src/elements/wordpress/wordpress.wc.svelte:133]: waiting for the merge in https://github.com/threefoldtech/info_grid/pull/3

File: src/App.svelte (hunk 2 above)
Comment (0oM4R) [reply src/App.svelte]: done

**Review by zaelgohary — APPROVED** [review]

**Pull request merged**